applications/concept-catalogue/src/main/kotlin/no/brreg/Generated.java

```diff
@@ -0,0 +1,11 @@
+package no.brreg;
```

Comment:
Må hele denne til for å få jacoco til å slutte å klage ?

Reply:
jacoco ignorerer det som er annotert med @Generated, så lenge den har RetentionPolicy CLASS eller RUNTIME. Det som ble anbefalt hos jacoco var å bruke lombok.Generated, men jeg følte det var veldig overkill å hente inn hele lombok bare for å få tak i en annotering.

-> Lagde egen @Generated